taskcluster/config.yml

```diff
@@ -3,6 +3,10 @@ trust-domain: taskcluster
 # this is lower priority than the decision task
 task-priority: high
 
+treeherder:
+  group-names:
+    'I': 'Docker Image Builds'
```

Comment:
is this needed if we don't have a treeherder integration?

Reply:
Yes it is. I want to talk to RelEng about this alongside some other changes I'd like to suggest after implementing these changes.